components/com_users/tmpl/remind/default.php

```diff
@@ -29,7 +29,7 @@
             </h1>
         </div>
     <?php endif; ?>
-    <form id="user-registration" action="<?php echo Route::_('index.php?option=com_users&task=remind.remind'); ?>" method="post" class="com-users-remind__form form-validate form-horizontal well">
+    <form id="user-registration" action="<?php echo Route::_('index.php?task=remind.remind'); ?>" method="post" class="com-users-remind__form form-validate form-horizontal well">
```

Comment:
```suggestion
    <form action="<?php echo Route::_('index.php?task=remind.remind'); ?>" method="post" id="user-registration" class="com-users-remind__form form-validate form-horizontal well">
```
action/method/name/id/class